app/utilities/Resources.java

```diff
@@ -0,0 +1,19 @@
+package utilities;
+
+/**
+ * Created by test on 02/02/16.
+ */
+public class Resources {
+    public static final String BAD_REQUEST = "Error in input data";
+    public static final String BAD_REQUEST_PASSWORD_MISMATCH = "Error in input data";
+    public static final String BAD_REQUEST_WRONG_CONFIRMATION_TOKEN = "Wrong confirmation token";
+    public static final String SERVER_NAME = "localhost:9000"; //"api.table4you" when deployed
+    public static final String VERSION = "v1.0";
+
+    public static boolean passwordCheck(UserHelper userHelper){
+        if (userHelper.password.equals(userHelper.passwordConfirmation))
+                return true;
```

Comment:
Please use brackets even if single statement, since it can lead to an error in further development process. This would also be good code formatting practice.